### PR TITLE
Translate to English

### PR DESCRIPTION
**Pull request by kant2002**
Only README.md, Changelog.md and simplified Qemu document translated

And Happy New Year!

### CODE REVIEW

File: ChangeLog_en.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+##### 20210831
+-Re-acquire the toolchain code and refactor the entire system
+-Remove the "-march=loongarch" compilation parameter, and set the default detection architecture of the tool chain.
```
Comment (sterling-teng) [inline ChangeLog_en.md:3]: tool chain -> toolchain

File: Qemu_For_LoongArch64-Simple_en.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+    The next step is to register the LoongArch executable file information in Binfmt, use the following command:
+
+```sh
+echo ":qemu-loongarch64:M::\x7fELF\x02\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\x02\x00\x02\x01:\xff\xff\ xff\xff\xff\xfe\xfe\x00\xff\xff\xff\xff\xff\xff\xff\xff\xfe\xff\xff\xff:/bin/qemu-loongarch64:"> /proc/sys/ fs/binfmt_misc/register
+```
+
+    The above command registers the "header information" of the LoongArch executable file, and specifies that the qualified files will be executed using the "/bin/qemu-loongarch64" command, so this "/bin/qemu-loongarch64" command must be true and effective.
+
+    If you want to know what Binfmt is, you can search on the Internet by yourself, or look at the brief description of the documents mentioned in the preface.
```
Comment (sterling-teng) [inline Qemu_For_LoongArch64-Simple_en.md:66]: prefacr -> Introduction

File: README_en.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    Corresponding CLFS system: [loongarch64-clfs-system-20210818](https://github.com/sunhaiyong1978/CLFS-for-LoongArch/releases/download/20210818/loongarch64-clfs-system-20210818.tar.bz2)
+
+    Cross tool chain: [loongarch64-clfs-20210812-cross-tools](https://github.com/sunhaiyong1978/CLFS-for-LoongArch/releases/download/20210812/loongarch64-clfs-20210812-cross-tools.tar.xz)
+  
+
+[CLFS_For_LoongArch64-20210812](https://github.com/sunhaiyong1978/CLFS-for-LoongArch/blob/main/CLFS_For_LoongArch64-20210812.md)：
+
+    Corresponding CLFS system: [loongarch64-clfs-system-20210812](https://github.com/sunhaiyong1978/CLFS-for-LoongArch/releases/download/20210812/loongarch64-clfs-system-20210812.tar.bz2)
+
+    Cross tool chain: [loongarch64-clfs-20210812-cross-tools](https://github.com/sunhaiyong1978/CLFS-for-LoongArch/releases/download/20210812/loongarch64-clfs-20210812-cross-tools.tar.xz)
```
Comment (sterling-teng) [inline README_en.md:34]: Cross compile tool chain

File: Qemu_For_LoongArch64_en.md [hunk 4]
```diff
@@ -0,0 +1,316 @@
+# <center>Use QEMU to run a Linux system based on the LoongArch64 architecture</center>
+
+<center>(Qemu For LoongArch64)</center>  
+
+<center>Author: Sun Haiyong</center>
+<center>Translator: Andrii Kurdiumov</center>
+
+## 1 Introduction
+    Loongson Zhongke launched a new instruction set architecture LoongArch in 2021. This 64-bit instruction set is called LoongArch64.
```
Comment (sterling-teng) [inline Qemu_For_LoongArch64_en.md:9]: Loongson Technology

File: CLFS_For_LoongArch64-20210903_en.md [hunk 5]
```diff
@@ -0,0 +1,3932 @@
...
+
+* By setting the "SYSDIR" variable to facilitate the use of the "base directory", this variable sets a specific directory as the "base directory", and the work related to this build is carried out in this directory.
+
+* The "downloads" directory is used to store source code packages and patch files of various software;
+
+* The "build" directory is used to compile various software packages;
+
+* The "cross-tools" directory is used to store cross compilation tool chains and related software;
+
+* "Sysroot" is used to store the target platform system.
```
Comment (sterling-teng) [inline CLFS_For_LoongArch64-20210903_en.md]: Sysroot -> sysroot

**Comment by sunhaiyong1978** [comment]
Hi kant2002,
确认一下sterling-teng提出的修改意见?

**Comment by kant2002** [comment]
@sunhaiyong1978 I do correct all of them.

**Comment by kant2002** [comment]
Thanks!

**Comment by sunhaiyong1978** [comment]
不客气，我应该感谢你。

**Pull request merged**